pgo/workloads/cs-example/run

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -eu
+
+trap 'pkill -P $$ || true' EXIT
+
+DIR=`dirname $(realpath "$0")`
+BIN=`realpath "$1"`
+TYPE="$2"
+OUT=`realpath "$3"`
+cd "$DIR"
+
+function wait_for_port {
+    timeout --foreground "$3" bash -c 'until printf "" 2>>/dev/null >>/dev/tcp/'"$1/$2"'; do sleep 0.1 || exit 1; done'
+}
+
+SERVER_ADDRS=(127.42.42.1 127.42.42.2 127.42.42.3)
```

Comment:
Let's randomize the subnet to prevent conflicts if two parallel compile are running (probably a bad idea with pgo, but still).

Reply:
Right. I wrote this with containers in mind, but apparently `dbuild` runs with `--network host` so this doesn't work even with containers.

But isn't there a way to prevent conflicts better than randomization? The number of loopback subnets isn't even that big...

Reply:
It's what we use elsewhere. I don't think there's a problem with that, the odds of two builds running simultaneously are low to begin with.